test/gateways/hosted_zone_gateway_test.py

```diff
@@ -0,0 +1,45 @@
+from src.gateways.hosted_zone_gateway import HostedZoneGateway
+from test.client_mock import ClientMock
+
+
+class TestHostedZoneGateway:
+    def test_correct_zone_is_found(self):
+        client = ClientMock()
+        hosted_zone_gateway = HostedZoneGateway(client)
+
+        assert hosted_zone_gateway.get_hosted_zone('test-blue.example.co.uk.') == {
+            'Id': '/hosted/id', 'Name': 'example.co.uk.'
+        }
+
+    def test_correct_zone_records_found(self):
+        client = ClientMock()
+        hosted_zone_gateway = HostedZoneGateway(client)
+
+        assert hosted_zone_gateway.get_hosted_zone_records('test-blue.example.co.uk.') == [
+            {
```

Comment:
This should be DRY'd out using a global variable because this data is shared with ClientMock

Reply:
Attempted this, we have an impure function somewhere which is modifying the return of client portal, so defining these as globals is breaking other tests